hw/ip_templates/clkmgr/doc/theory_of_operation.md.tpl

```diff
@@ -1,4 +1,4 @@
-# Theory of Operation
+<%text filter="n"># Theory of Operation
```

Comment:
Are you doing this to avoid mako from interpreting ## as a comment? I tend to replace such by ${"##"} instead. The %text directive ends up affecting the full file and ends requiring placing </%text> around specific spots, so it over-reaches and can create confusion if they are not correctly paired. By the way, a single '#' is not a problem.

Reply:
Yes, and I didn't much feel like going and modifying a ton of doc lines for this PR.

Personally, I'd just use HTML header tags (which is still markdown syntax) instead of markdown's special header syntax wrapped in a mako escape. Then mako won't get in the way.

Reply:
It looks like the convention has been to wrap the header alone in the text directive (which I forgot from https://github.com/lowRISC/opentitan/pull/19157 ). I'll make that change.

If you'd like to change that style convention, I'd recommend raising it with the folks who have done focused work on maintaining the websites.